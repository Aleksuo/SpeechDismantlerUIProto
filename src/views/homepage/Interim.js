import React from 'react'
import {Paper, Typography} from '@material-ui/core'
import PropTypes from 'prop-types'

const Interim = ({ interim }) => {
    return (
        <div>
            <Paper elevation={2} style={{ color: "gray", height: "5vh", textAlign: 'center', overflow: "auto" }}>
                <Typography color="inherit">{interim}</Typography>
            </Paper>
        </div>
    )
}

Interim.propTypes = {
    interim: PropTypes.string
}

export default Interim