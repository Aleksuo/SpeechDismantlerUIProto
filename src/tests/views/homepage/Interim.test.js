/* eslint-disable */
import React from "react"
import Enzyme, {shallow, mount,render} from "enzyme"
import Interim from "../../../views/HomePage/Interim"
import Adapter from "enzyme-adapter-react-16"

Enzyme.configure({adapter: new Adapter()})

describe("Interim component", () =>{
    test("renders", () =>{ 
        const wrapper = shallow(<Interim interim={""}/>) 
        expect(wrapper).toMatchSnapshot()
    })
})