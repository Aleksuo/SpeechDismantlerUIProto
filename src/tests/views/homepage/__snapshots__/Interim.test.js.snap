// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Interim component renders 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Interim
    interim=""
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <WithStyles(Paper)
        elevation={2}
        style={
          Object {
            "color": "gray",
            "height": "5vh",
            "overflow": "auto",
            "textAlign": "center",
          }
        }
      >
        <WithStyles(Typography)
          color="inherit"
        >
          
        </WithStyles(Typography)>
      </WithStyles(Paper)>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <WithStyles(Typography)
          color="inherit"
        >
          
        </WithStyles(Typography)>,
        "elevation": 2,
        "style": Object {
          "color": "gray",
          "height": "5vh",
          "overflow": "auto",
          "textAlign": "center",
        },
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": "",
          "color": "inherit",
        },
        "ref": null,
        "rendered": "",
        "type": [Function],
      },
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <WithStyles(Paper)
          elevation={2}
          style={
            Object {
              "color": "gray",
              "height": "5vh",
              "overflow": "auto",
              "textAlign": "center",
            }
          }
        >
          <WithStyles(Typography)
            color="inherit"
          >
            
          </WithStyles(Typography)>
        </WithStyles(Paper)>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <WithStyles(Typography)
            color="inherit"
          >
            
          </WithStyles(Typography)>,
          "elevation": 2,
          "style": Object {
            "color": "gray",
            "height": "5vh",
            "overflow": "auto",
            "textAlign": "center",
          },
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "",
            "color": "inherit",
          },
          "ref": null,
          "rendered": "",
          "type": [Function],
        },
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
